### PR TITLE
Add tests to verify user-modify is unsupported

### PR DESCRIPTION
**Pull request by howard-e**
This adds tests to check that prefixed and unprefixed values of `user-modify` is unsupported.

### CODE REVIEW

**Comment by zcorpan** [comment]
Per https://developer.mozilla.org/en-US/docs/Web/CSS/user-modify#specifications this property was initially in the CSS UI spec. So the correct place for these tests is something like `/css/css-ui/historical/`

File: user-modify/deprecated-webkit-user-modify-03.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        height: 10px;
+        background-color: red;
+    }
+}
+</style>
+<div id="content"></div>
+<script>
+    test(function() {
+        const content = document.getElementById('content');
+        assert_not_equals(content.style.backgroundColor, 'rgb(255, 0, 0)');
```
Comment (zcorpan) [inline user-modify/deprecated-webkit-user-modify-03.html:19]: This should use `getComputedStyle` instead.

**Comment by zcorpan** [comment]
Also remove `deprecated-` from the filenames since it's not technically deprecated. If it's in a `historical` directory it's clear I think.

File: user-modify/deprecated-webkit-user-modify-03.html (hunk 1 above)
Comment (howard-e) [reply user-modify/deprecated-webkit-user-modify-03.html]: Thanks! Updated the test

**Comment by howard-e** [comment]
@zcorpan thanks for the feedback! I've addressed your points.

**Comment by zcorpan** [comment]
The lint check is failing because of lack of `<link rel="help" href="spec-link-url">` which, in this case, maybe the appropriate link is https://developer.mozilla.org/en-US/docs/Web/CSS/user-modify since it's non-standard.

I see now that Gecko supports `-moz-user-modify`. You should test that also, and unprefixed for completeness.

File: css/css-ui/historical/moz-user-modify-01.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>-moz-user-modify should be unsupported in author origin</title>
+<link rel="help" href="https://developer.mozilla.org/en-US/docs/Web/CSS/user-modify">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+#modified {
+    -moz-user-modify: read-write-plaintext-only;
```
Comment (zcorpan) [inline css/css-ui/historical/moz-user-modify-01.html:9]: Gecko doesn't support this value, maybe test with `read-write` instead.
```suggestion
    -moz-user-modify: read-write;
```

File: css/css-ui/historical/user-modify-01.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>-user-modify should be unsupported in author origin</title>
```
Comment (zcorpan) [inline css/css-ui/historical/user-modify-01.html:3]:
```suggestion
<title>user-modify should be unsupported in author origin</title>
```

File: css/css-ui/historical/user-modify-01.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+<link rel="help" href="https://developer.mozilla.org/en-US/docs/Web/CSS/user-modify">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+#modified {
+    user-modify: read-write-plaintext-only;
```
Comment (zcorpan) [inline css/css-ui/historical/user-modify-01.html:9]:
```suggestion
    user-modify: read-write;
```

File: css/css-ui/historical/user-modify-02.html [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>-user-modify should be unsupported in contenteditable</title>
```
Comment (zcorpan) [inline css/css-ui/historical/user-modify-02.html:3]:
```suggestion
<title>user-modify should be unsupported in contenteditable</title>
```

**Review by zcorpan — APPROVED** [review]
LGTM after nits are fixed.

File: css/css-ui/historical/moz-user-modify-01.html (hunk 2 above)
Comment (howard-e) [reply css/css-ui/historical/moz-user-modify-01.html]: Good catch, I've addressed this feedback

**Comment by zcorpan** [comment]
Test results for this PR:

https://wpt.fyi/results/css/css-ui/historical?label=pr_head&max-count=1&pr=35471

**Pull request merged**